python/lsst/meas/deblender/baseline.py

```diff
@@ -632,24 +751,54 @@ def computeImage(self, cx, cy):
 
 def _fitPsfs(fp, peaks, fpres, log, psf, psffwhm, img, varimg,
              psfChisqCut1, psfChisqCut2, psfChisqCut2b,
-             **kwargs):
-    """!
-    Fit a PSF + smooth background models to a small region around each
-    peak (plus other peaks that are nearby) in the given list of
-    peaks.
-
-    @param[in]      fp           Footprint containing the Peaks to model
-    @param[in]      peaks        a list of all the Peak objects within this Footprint
-    @param[in,out]  fpres        a PerFootprint result object where results will be placed
-    @param[in,out]  log          pex Log object
-    @param[in]      psf          afw PSF object
-    @param[in]      psffwhm      FWHM of the PSF, in pixels
-    @param[in]      img          the Image in which this Footprint finds itself
-    @param[in]      varimg       Variance plane
-    @param[in]      psfChisqCut* floats: cuts in chisq-per-pixel at which to accept
-                                 the PSF model
-
-    Results go into the fpres.peaks objects.
+             **kwargs
+             ):
+    """
+    Fit a PSF + smooth background model (linear) to a small region around each peak.
+    This routine uses a linear least squares algorithm to fit each peak (and neighboring peaks)
```

Comment:
I think we want a new line between the "short summary" (first line) and "extended description" (the rest of this text).